### PR TITLE
feat: consume the new search bar

### PR DESCRIPTION
**Pull request by vikrantgupta25**
### Summary

this pr adds the consumption for the new search bar v2 in the logs explorer page 

#### Related Issues / PR's

contributes to https://github.com/SigNoz/signoz/issues/5150

#### Screenshots

NA

### CODE REVIEW

File: frontend/src/container/QueryBuilder/filters/QueryBuilderSearchV2/QueryBuilderSearchV2.tsx [hunk 1]
```diff
@@ -324,6 +328,21 @@ function QueryBuilderSearchV2(
 
 				if (isMulti) {
 					const { tagKey, tagOperator, tagValue } = getTagToken(searchValue);
+					console.log(tagValue, value);
```
Comment (nityanandagohain) [inline frontend/src/container/QueryBuilder/filters/QueryBuilderSearchV2/QueryBuilderSearchV2.tsx:331]: remove

**Review by nityanandagohain — CHANGES_REQUESTED** [review]
Please add test cases to alteast start in that direction, else it will be treated as a secondary citizen and these manual testing will get monotonous.

Checks
- [ ]  static field data type is not shown eg:- severity_text, severity_number 
- [ ] Not able to update the value of operator after running a query ( replaces with old value) ( change the value wait for the suggestion to update then it works but rapid typing doesn't)
- [x] Contains and other filters working properly
- [ ] Filtering from the detail page by clicking on attribute name not working
- [ ] Filter in , filter out from detail page doesn't update the search bar. (refreshing the window makes them appear)
- [ ] There is a flicker after selecting a key value pair.
- [ ] The filter doesn't get cleared if I got to some other page like alerts and come back

Questions ?
* if attr named `x` is of both float and string, then if I type x then is string type selected by default ( that what it seems to be the case)?

Suggestions
* the tag/resource infor feels too far to the right
* the suggestions is scrollable ( the user won't know unless they look extreme right)

File: frontend/src/container/QueryBuilder/filters/QueryBuilderSearchV2/QueryBuilderSearchV2.tsx (hunk 1 above)
Comment (vikrantgupta25) [reply frontend/src/container/QueryBuilder/filters/QueryBuilderSearchV2/QueryBuilderSearchV2.tsx]: done

**Comment by vikrantgupta25** [comment]
@nityanandagohain fixed the 2nd point as discussed, remaining will move to separate issues

**Review by nityanandagohain — DISMISSED** [review]
lgtm, please create issue for test and keep it in high priority so that we don't need to go through these tests again for one of the next pr's

**Comment by vikrantgupta25** [comment]
@nityanandagohain also addressed the same name different data type issue. if no operator is entered until we show both and do not tokenise and once the operator comes we then pick on priority.

**Comment by vikrantgupta25** [comment]
created one P0 task on my name for the test cases - https://github.com/SigNoz/engineering-pod/issues/1724 cc @nityanandagohain

**Review by YounixM — APPROVED** [review]

**Review by nityanandagohain — APPROVED** [review]
LGTM, @srikanthccv do skim through the feature flag part once if somethng looks suspecious/missing, I have less context on it.

**Pull request merged**